ezyrb/ann.py

```diff
@@ -0,0 +1,83 @@
+"""
+Module for Artificial Neural Network (ANN) Prediction.
+"""
+import torch
+import torch.nn as nn
+import numpy as np
+from .approximation import Approximation
+
+class ANN(Approximation):
+    """
+    Feed-Forward Artifical Neural Network (ANN).
+    : param int trained_epoch: number of already trained iterations.
```

Comment:
the space are important in the documentation style. Here is :param without spaces. Same in the following lines.

Reply:
Thanks, I will fix that.